common/config/rollup/rollup.config.js

```diff
@@ -18,5 +19,5 @@ export default (packageJson) => ({
     format: 'cjs',
     sourcemap: true
   },
-  plugins: [commonjs(), sourcemaps(), svg(), json({ compact: true })]
+  plugins: [commonjs(), sourcemaps(), svg(), json({ compact: true }), nodeResolve({browser: true})]
```

Comment:
@JamesBurnside I had to change the rollup.config.js to use uuid because Rollup could not tell which uuid library I wanted to import. I got this solution from here https://stackoverflow.com/questions/59937658/importing-uuid-in-rollup-esm-library-creates-an-import-statement-for-crypto-in. Not sure if it's the right way to go.

Reply:
Oh thanks for flagging this, let me take a look tomorrow before completing this